scripts/notebook-normalizer/qiskit_docs_notebook_normalizer/cell_output_data.py

```diff
@@ -52,6 +53,16 @@ def remove_circuit_drawing_html(output_data: dict) -> bool:
             return True
     return False
 
+def remove_inline_katex_expression(output_data: dict) -> bool:
+    """Convertes inline katex expressions into display mode if needed and returns True if any changes were made."""
+    if latex := output_data.get("text/latex"):
+        if latex.startswith('$\\displaystyle') and latex.endswith('$'):
+            latex = latex.replace('$\\displaystyle', '$$\n') # Replace the first '$'
+            latex = re.sub(r'\$$', '\n$$', latex) # Replace the last '$'
+            output_data["text/latex"] = latex
```

Comment:
Is starting with `\\displaystyle` guaranteed? If not, I think it's safe enough that we're in the `text/latex` output and have starting and ending `$`.

Also might be even safer to strip the output too:

```suggestion
    if latex := output_data.get("text/latex").strip():
        if latex.startswith('$\\displaystyle') and latex.endswith('$'):
            latex = latex.replace('$\\displaystyle', '$$\n') # Replace the first '$'
            latex = re.sub(r'\$$', '\n$$', latex) # Replace the last '$'
            output_data["text/latex"] = latex
```